src/lib/API/api.js

```diff
@@ -487,6 +487,41 @@ export class APIService {
     return this.sharedClient.post(url, payload, options)
   }
 
+  async getOTPLEvents(sender, chainId, otpAddress, from) {
+    // if (chainId === NETWORK_ID.FUSE) {
+
+    // } else {
+    const explorer = Config.ethereum[chainId].explorerAPI
+    log.info('polltest -- getOTPLEvents -->', { sender, chainId, otpAddress, from, explorer })
+    const otplLogs = await this.sharedClient.get('/api', {
+      params: {
+        module: 'logs',
+        action: 'getLogs',
+        fromBlock: from - 10000,
```

Comment:
why? same logic as getting txs.

Reply:
what from above is different then the query done for txs?
the reason I (believe) to need a separate flow is because of the tatum being used in the tx's flow

I cannot find an endpoint in their docs which gives back the events we need

Reply:
I am not sure but need to retest, as the weird thing is that when I set offset to 10k, it returns empty.
If I do the from minus, it gave back the data requested?

Reply:
1. compare and find the differences
2. if the endpoint isnt working use the regular explorer apis

Reply:
2. thats what this uses? (regular explorer api